src/reason-parser/reason_pprint_ast.ml

```diff
@@ -1747,8 +1747,23 @@ let partitionFinalWrapping listTester wrapFinalItemSetting x =
         else
           Some (List.rev revEverythingButLast, last)
 
+let should_preserve_requested_braces expr =
+  let {literalAttrs} = partitionAttributes expr.pexp_attributes in
+  match expr.pexp_desc with
+  | Pexp_ifthenelse _
+  | Pexp_try _ -> false
+  | _ -> Reason_attributes.has_preserve_braces_attrs literalAttrs
+
 let semiTerminated term = makeList [term; atom ";"]
 
+let makeBreakableBlock ?(inline=false) letItems =
```

Comment:
I'm not a huge fan of abstractions on top of `makeList`. The exact behaviour of `makeBreakableBlock` is very hard to predict without inspecting its implementation.
`makeBreakableBlock` seems to be used only once too. Is there a reason why you didn't use `makeList` with all named args in the letList printing? Just using `makeList` with all named args makes it very clear what is going to happen.